Source/CarthageKit/NetworkClient.swift

```diff
@@ -0,0 +1,54 @@
+import ReactiveCocoa
+
+
+public func createURLRequest(URL: NSURL, _ headers: [String: String] = [:]) -> NSURLRequest {
+	let urlRequest = NSMutableURLRequest(URL: URL)
+	urlRequest.allHTTPHeaderFields = headers
```

Comment:
As long as you explicitly tested that the 2 are equivalent, I prefer what you wrote in the PR.